pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -16599,7 +16599,7 @@ spec:
 // the ResolutionRequest's name is generated by resolverName, namespace and runName.
 func getResolvedResolutionRequest(t *testing.T, resolverName string, resourceBytes []byte, namespace string, runName string) resolutionv1beta1.ResolutionRequest {
 	t.Helper()
-	name, err := remoteresource.GenerateDeterministicName(resolverName, namespace+"/"+runName, nil)
+	name, err := remoteresource.GenerateDeterministicName(resolverName, namespace+"/"+runName, nil, "")
```

Comment:
Instead of requiring passing in required nil / "" values, can we use variadic options instead?

Reply:
done!